src/mumble/VoiceRecorder.cpp

```diff
@@ -311,7 +311,8 @@ void VoiceRecorder::run() {
 			break;
 		}
 
-		while (!m_abort && !m_recordBuffer.isEmpty()) {
+		const bool shouldMixDown = m_config.mixDownMode && m_config.transportEnable;
```

Comment:
Mixdown mode sounds like it is a different type than boolean. Maybe an explicit comparison makes things more future-proof?

Reply:
`m_config.mixDownMode` is indeed defined as a `bool`. I can explicitly compare to `true` though in this case because I agree :)

Reply:
Nah, if it's a `bool`, we can keep it as it is. I just figured it might be something like an enum :shrug:

Reply:
Sounds good, I'll include this in the final adjustments